bluetooth: Open ble defconfig for openvela ble stack.

zhongzhijie1 opened the pull request:
1. ble gattc/gatts.
2. gap: adv/scan leconnect & smp.
3. no need use of VENDOR_NONE: default set to 'y' in Kconfig.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/open-vela/vendor_openvela?pullRequest=3) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/open-vela/vendor_openvela?pullRequest=3) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/open-vela/vendor_openvela?pullRequest=3) it.</sub>

pengxianghao21 commented:
invalid PR

zhongzhijie1 committed:
bluetooth: Open ble defconfig for openvela ble stack.

bug: v/63786

1. ble gattc/gatts.
2. gap: adv/scan leconnect & smp.
3. no need use of VENDOR_NONE: default set to 'y' in Kconfig.

Signed-off-by: zhongzhijie1 <zhongzhijie1@xiaomi.com>
